src/index.js

```diff
@@ -1,51 +1,62 @@
 /** @format */
 
-/** @format */
-
-// import _ from 'lodash';
 import './style.css';
+import setItem from './asset/js/storage.js';
+import {
+  addList, input,
+} from './asset/js/add.js';
+/* eslint-disable */
```

Comment:
- [ ] as the previous CR already mentioned please do not disable Eslint it will help check for your Javascript code standards. Kindly enable Eslint and fix the errors manually.